### PR TITLE
Remove scroll:auto on oncoprint because it is causing various problem…

### PR DESCRIPTION
**Pull request by alisman**
I made a change to oncoprint css to hide scrollbars when they were not needed (no overflow).  This caused various problems in different browsers so I am reverting the cosmetic change.

# What? Why?
Fix # .

Changes proposed in this pull request:
- a
- b

# Checks
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). If you are part of the
cBioPortal organization, notify the approprate team (remove inappropriate):

@cBioPortal/frontend

If you are not part of the cBioPortal organization look at who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them here:

### CODE REVIEW

**Comment by adamabeshouse** [comment]
I cant find it now but, wont this bring back the issue you were fixing, with the vertical scrollbar not appearing or something like that?

File: end-to-end-tests/wdio/webdriver-manager.conf.js [hunk 1]
```diff
@@ -35,18 +35,14 @@ exports.config = {
     // and 30 processes will get spawned. The property handles how many capabilities
     // from the same test should run tests.
     //
-    maxInstances: 1,
+    maxInstances: 5,
```
Comment (adamabeshouse) [inline end-to-end-tests/wdio/webdriver-manager.conf.js]: :+1:

**Comment by alisman** [comment]
@adamabeshouse no, this is reversion to the way the scrolling initially was: forced-on.  I don't like it when the scrollbars show when no scrolling is actually possible, but that seems to be the most robust way of solving all these various subtle but horrible behavior inconsisteencies.

File: end-to-end-tests/wdio/webdriver-manager.conf.js (hunk 1 above)
Comment (inodb) [reply end-to-end-tests/wdio/webdriver-manager.conf.js]: @alisman Wondering how this relates to number of cores on physical machine.

**Comment by alisman** [comment]
@adamabeshouse I think we reached conclusion that commits don't always have to be squashed if they actually represent different tasks/updates and we want that preserved in commit history.  In this case they do, although I think the screenshot updates should probably be merged with the updates that necessitated them.  I will do that.

**Comment by alisman** [comment]
squashing done!

**Pull request merged**